scripts/helpers.mjs

```
@@ -64,6 +64,28 @@ export const ifAny = function () {
+// {{/ifAll}}
+
+export const ifAll = function() {
+  // Convert arguments to an array, excluding the last 'options' object
+  var args = Array.prototype.slice.call(arguments, 0, arguments.length - 1);
+  var options = arguments[arguments.length - 1];
+
+  // Check if every argument is truthy
+  var allTruthy = args.every(Boolean);
+
+  if (allTruthy) {
+      return options.fn(this);
```

ESLint

[error] Expected indentation of 4 spaces but found 6.

(indent)